app/components/Modal/WithdrawModalNew.jsx

```diff
@@ -767,9 +767,16 @@ class WithdrawModalNew extends React.Component {
             descriptor = memo ? new Buffer(memo, "utf-8") : "";
             to = btsAccount.get("id");
         } else {
-            assetName = gatewayStatus.useFullAssetName
-                ? selectedGateway.toLowerCase() + "." + assetName
-                : assetName;
+            if (
+                assetName.toLowerCase() === "monero" &&
```

Comment:
I don't like this change as the new modals aim to be generalized for making it easier to add new gateways. I would rather have this done on a per gateway in the settings file. 

Something like
``` assetWithdrawlAlias: { monero: xmr } ```